core/src/test/java/bio/terra/pearl/core/service/kit/LivePepperDSMClientTest.java

```diff
@@ -65,19 +65,46 @@ void startup(TestInfo info) throws Exception {
 
     @Transactional
     @Test
-    public void testMalformedErrorResponseFromPepper() throws Exception {
+    public void testBadErrorResponseFromPepper() throws Exception {
         // Arrange
-        var unexpectedJsonBody = "{\"error\": \"boom\"}";
+        var unexpectedJsonBody = "{\"unexpected\": \"boom\"}";
         mockPepperResponse(HttpStatus.BAD_REQUEST, unexpectedJsonBody);
 
         // "Act"
-        Executable act = () -> client.sendKitRequest(enrollee, kitRequest, address);
+        Executable act = () -> client.sendKitRequest("testStudy", enrollee, kitRequest, address);
 
         // Assert
         PepperException pepperException = assertThrows(PepperException.class, act);
         assertThat(pepperException.getMessage(), containsString(unexpectedJsonBody));
     }
 
+    @Transactional
+    @Test
+    public void testErrorResponseFromPepperWithUnexpectedAttributes() throws Exception {
```

Comment:
I got tripped up on new attributes I hadn't yet configured causing exceptions, which would also be bad to have happen in production.